packages/suite-web/e2e/tests/onboarding/t2t1-recovery-fail.test.ts

```diff
@@ -6,12 +6,12 @@ describe('Onboarding - recover wallet T2T1', () => {
         cy.task('startBridge');
         cy.viewport(1080, 1440).resetDb();
         cy.prefixedVisit('/');
-        cy.task('startEmu', { wipe: true, version: '2.4.3' });
```

Comment:
if there is no good reason to test specific version, we should always  test the latest one